profiler/README.MD

```diff
@@ -1,18 +1,5 @@
 ﻿# dd-trace-dotnet/profiler
 
-This repo ([dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet)) contains the sources for the client-side components of the Datadog product suite for Application Telemetry Collection and Application Performance Monitoring for .NET Applications.
-
-This directory (`/profiler`) contains the source for the __Continuous Profiler for .NET__ product, which is a part of that product suite. 
-
-### Copyright
-
-Copyright (c) 2020 Datadog
-[https://www.datadoghq.com](https://www.datadoghq.com/)
-
-### License
-
-See [license information](../LICENSE).
-
 ## Note
 
 Currently, this directory is empty.
```

Comment:
Github won't let me comment below, but https://github.com/DataDog/dd-continuous-profiler-dotnet is a Datadog internal repository so we probably shouldn't be linking to it.

Reply:
And mentions of "Profiler" below should say "Continuous Profiler".